app/services/idv/steps/in_person/state_id_step.rb

```diff
@@ -37,7 +35,7 @@ def call
            end
           end
 
-          maybe_redirect_to_verify_info(flow_session[steps[:address].to_s].blank?)
+          redirect_to idv_in_person_verify_info_url if flow_session[steps[:address].to_s]
```

Comment:
```suggestion
          redirect_to idv_in_person_verify_info_url if flow_session['Idv::Steps::InPerson::AddressStep']
```
Using the [STEPS array](https://github.com/18F/identity-idp/blob/3507d711066c302984894e488a527a6083bd22d2/app/services/idv/flows/in_person_flow.rb#LL8C4-L8C4) here is going to cause trouble when you remove the address step, so I recommend changing this now.

There's an ongoing conversation ([example JIRA ticket](https://cm-jira.usa.gov/browse/LG-9983)) on how to track the user's progress outside the flow state machine. We're using fields on [Idv::Session](https://github.com/18F/identity-idp/blob/main/app/services/idv/session.rb) and methods in [IdvSession concern](https://github.com/18F/identity-idp/blob/main/app/controllers/concerns/idv_session.rb), but it needs solidifying/refactoring.

Reply:
Done. commit 69101356c159c13f913e8ff762d8b2139e3da6dd